lib/main.coffee

```diff
@@ -96,12 +119,11 @@ module.exports =
         for grammar in pack.grammars
           grammars[grammar.scopeName] = grammar
 
-        for editor in atom.workspace.getTextEditors()
-          do (editor) ->
-            grammar = editor.getGrammar()
-            if grammar.packageName is packName
-              debug and console.log 'Updating grammar for editor: ', editor
-              editor.setGrammar grammars[grammar.scopeName]
+        atom.workspace.getTextEditors().forEach (editor) ->
```

Comment:
I don't agree with this change

Reply:
It's a nitpick either way, so you get precedence here.

I'll change it back to a `for` loop, but the `do (editor) ->` part is unnecessary.